_includes/recur.md

```diff
@@ -60,6 +66,7 @@ Content-Type: application/json
 | {% icon check %} | └─➔&nbsp;`productCategory`     | `string`     | A product category or number sent in from the payee/merchant. This is not validated by Swedbank Pay, but will be passed through the payment process and may be used in the settlement process.                                                                                        |
 | {% icon check %} | └─➔&nbsp;`orderReference`      | `String(50)` | The order reference should reflect the order reference found in the merchant's systems.                                                                                                                                                                                               |
 | {% icon check %} | └─➔&nbsp;`subsite`             | `String(40)` | The subsite field can be used to perform [split settlement][split-settlement] on the payment. The subsites must be resolved with Swedbank Pay [reconciliation][settlement-and-reconciliation] before being used.                                                                      |
+|                  | └➔&nbsp;`metadata`             | `object`      | The keys and values that should be associated with the payment. Can be additional identifiers and data you want to associate with the payment.                                                                                                                                                 |
```

Comment:
Should we have a section in "other features" explaining `metadata` in more detail, perhaps?

Reply:
We already have one in all payments and checkout + payment menu :)

Reply:
Then let's link to it from the field description?

Reply:
fixed :)